generated/nidigital/session.py

```diff
@@ -2867,13 +2876,13 @@ def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, time
             start = i * actual_samples_per_waveform
             end = start + actual_samples_per_waveform
             if sys.version_info.major >= 3:
-                measurements.append(Measurement(data=mv[start:end], site=site_list[i]))
+                waveforms.append(Waveform(site=site_list[i], data=mv[start:end]))
             else:
                 # memoryview in Python 2 doesn't support numeric types, so we copy into an array.array to put in the wfm. :( You should be using Python 3!
```

Comment:
@MrDuff8 when can we drop Python 2.7?

Reply:
The earliest will be when the Python org drops it, which is 1/1/20.